index.bs

```diff
@@ -150,7 +213,31 @@ the following steps:
 1. Return |encodedOutput|.
 
 <h3 algorithm id="delivery-time">Establishing report delivery time</h3>
-TODO
+The <dfn>report delivery time</dfn> for an [=impression=] |impression| and a
+[=conversion/conversion time=] |conversionTime| is the result of the following steps:
+1. Let |conversionTimeAfterImpression| be the difference between the
+    [=conversion/conversion time=] and [=impression/impression time=].
+2. Let |expiryDelta| be the difference between the [=impression/expiry=] and
+    the [=impression/impression time=]
+
+<dl class="switch">
+<dt>If |conversionTimeAfterImpression| <= (2 days - 1 hour)</dt>
```

Comment:
What you have is ok, but I'd usually put the `<dl>` inside a numbered step:

```suggestion
1. If:
    <dl class="switch">
    <dt>|conversionTimeAfterImpression| <= (2 days - 1 hour)</dt>
```
And similarly indented through the rest of the `<dl>`.

Reply:
Done